lib/pure/httpcore.nim

```diff
@@ -43,7 +43,9 @@ type
                       ## for specified address.
     HttpConnect       ## Converts the request connection to a transparent
```

Comment:
Fixed, sorry. I don't think commas are needed when listing enums on more than one line, but I had missed this style was adopted here